lib/ReactViewModels/MouseCoords.ts

```diff
@@ -196,10 +196,9 @@ export default class MouseCoords {
     const coordinates = Cartographic.fromDegrees(
       latLng.lng,
       latLng.lat,
-      undefined,
+      0,
       scratchCartographic
     );
-    coordinates.height = <any>undefined;
```

Comment:
Setting `height` as `undefined` resulted in some computations producing `NaN` values which broke the rectangle drawing.